### DISCUSSION TITLE
Duda Sobre Las Practicas

### DISCUSSION THREAD

**Discussion by craonay** [Launch-X-Latam/MisionFrontEnd#474] [Team Posts]
¿El método para entregar las practicas es solo subiendo las al repositorio propio?Hay que vincularlas a otro sitio o algo parecido o solo es subir las practicas al repositorio propio

**Comment by NoirOrange** [comment]
Solamente subelas al repositorio de tu perfil ;)

**Comment by Amhedriel** [comment]
Al perfil que tienes vinculado a Innovacción.
